.gitlab-ci.yml

```diff
@@ -43,6 +43,7 @@ stages:
     # Give $PRIMARY_USER permission to create cgroups
     - test/for_each_of_my_cgroups.sh chgrp $PRIMARY_USER
     - test/for_each_of_my_cgroups.sh chmod g+w $PRIMARY_USER
+    - apt install -y libsystemd-dev python-dev
```

Comment:
TODO for me: Add packages to `Dockerfile` on `main` and publish new Docker images.

Also, is there a specific reason why the `python3-pystemd` package was not used?

Reply:
Done in 45c81500f4901be8527a690600377ed97efb15e8.